proxy/IPAllow.cc

```diff
@@ -60,6 +64,29 @@ bwformat(BufferWriter &w, Spec const &spec, YAML::Mark const &mark)
 
 } // namespace swoc
 
+namespace YAML
+{
+template <> struct convert<ts::TextView> {
+  static Node
+  encode(ts::TextView const &tv)
```

Comment:
encode and decode don't seem to be called from anywhere on this PR.

Reply:
Good catch, thanks! 

the `encode()` and `decode()` are from the parent of the incoming changes that shouldn't be merged in. Removed the functions and the corresponding includes.